website/blog/2021-11-18-algolia-docsearch-migration/index.md

```diff
@@ -0,0 +1,117 @@
+---
+title: DocSearch migration
+authors:
+  - name: Clément Vannicatte
+    title: Software Engineer @ Algolia
+    url: https://github.com/shortcuts
+    image_url: https://github.com/shortcuts.png
+    twitter: sh0rtcts
+  - key: slorber
+
+tags: [search]
+image: ./img/social-card.png
+---
+
+[DocSearch](https://docsearch.algolia.com/) is migrating to a new, more powerful system.
+
+Docusaurus site owners should upgrade their site configuration **by February 1, 2022**
```

Comment:
Not sure it's worth it but still suggesting.
```suggestion
Docusaurus site owners with Docsearch enabled should upgrade their site configuration **by February 1, 2022**
```

Reply:
Probably quite implicit imho 😅